scripts/ci/gitlab/pipeline/check.yml

```diff
@@ -73,7 +73,8 @@ check-try-runtime:
             --runtime ./target/release/wbuild/"$NETWORK"-runtime/target/wasm32-unknown-unknown/release/"$NETWORK"_runtime.wasm \
             -lruntime=debug \
             --chain=${NETWORK}-dev \
-            on-runtime-upgrade live --uri wss://${NETWORK}-try-runtime-node.parity-chains.parity.io:443
+            on-runtime-upgrade --checks live \
+            --uri wss://${NETWORK}-try-runtime-node.parity-chains.parity.io:443
```

Comment:
```suggestion
            on-runtime-upgrade --checks \
            live --uri wss://${NETWORK}-try-runtime-node.parity-chains.parity.io:443
```

Reply:
So I will switch to `--checks pre-and-post` to keep the CI fast. Is that okay for you?